Orange/preprocess/preprocess.py

```diff
@@ -572,26 +572,42 @@ def __call__(self, data):
 
 class RemoveSparse(Preprocess):
     """
-    Remove sparse  features. Sparseness is determined according to
-    user-defined treshold.
+    Filter out the features with too many nan's or 0. Threshold is user defined.
 
     Parameters
     ----------
-    threshold : float
-        Minimal proportion of non-zero entries of a feature
+    threshold: int or float
+        if >= 1, the argument represents the allowed number of 0s or NaNs;
+        if below 0, it represents the allowed proportion of 0s or NaNs
+    filter0: bool
+        if True (default), preprocessor counts 0s, otherwise NoNs
     """
-
-    def __init__(self, threshold=0.05):
+    def __init__(self, threshold=5, filter0=True):
```

Comment:
I suggest to leave the default value at `threshold=0.05` so it remains backwards compatible.
(this does not impact the widget as it always calls it with a defined threshold anyway)